domains/noto/pkg/types/abi.go

```diff
@@ -124,3 +124,13 @@ type NotoTransferMetadata struct {
 	ApprovalParams       ApproveExtraParams    `json:"approvalParams"`       // Partial set of params that can be passed to the "approveTransfer" method to approve another party to perform this transfer
 	TransferWithApproval NotoPublicTransaction `json:"transferWithApproval"` // The public transaction that would need to be submitted by an approved party to perform this transfer
 }
+
+type BalanceOfParam struct {
+	Account string `json:"account"`
+}
+
+type BalanceOfResult struct {
+	TotalBalance string `json:"totalBalance"`
+	TotalStates  string `json:"totalStates"`
+	Overflow     string `json:"overflow"`
```

Comment:
Shouldn't these types be things like `HexUint256` and boolean, rather than all strings?